third_party/amd/lib/TritonAMDGPUToLLVM/OptimizeLDSUtility.cpp

```diff
@@ -57,8 +57,8 @@ Attribute createTmpLayout(Attribute layout, ArrayRef<unsigned> warpsPerCTA) {
         src.getMDim(), src.getNDim(), src.getIsTransposed(),
         src.getCTALayout());
   if (auto src = dyn_cast<triton::gpu::AMDWmmaEncodingAttr>(layout))
-    return triton::gpu::AMDWmmaEncodingAttr::get(ctx, warpsPerCTA,
-                                                 src.getCTALayout());
+    return triton::gpu::AMDWmmaEncodingAttr::get(
+        ctx, /* version = */ 1, warpsPerCTA, src.getCTALayout());
```

Comment:
Super nit: I think typically we do `/*version=*/` tightly without spaces in between.